data/mods/DinoMod/monsters/zinosaur_burned.json

```
@@ -0,0 +1,950 @@
+[
+  {
+    "type": "MONSTER",
+    "id": "mon_zilophosaurus_scorched",
+    "name": { "str": "scorched dilophosaurus zombie" },
+    "copy-from": "mon_zilophosaurus",
+    "description": "Heavily burned zombie dinosaur predator that still reeks of charred flesh.  Its flesh has mended into a leathery shell.",
```

```suggestion
    "description": "Heavily-burned zombie dinosaur predator that still reeks of charred meat.  Its flesh has mended into a leathery shell.",
```
Alternatively, substitute "skin" for the second sentence's use of "flesh".